docs/examples.rst

```diff
@@ -0,0 +1,21 @@
+.. _examples:
+
+Examples of interactions with the Hugging Face Hub
+==================================================
+
+- Creating the Model Card:
+    `Here <https://github.com/skops-dev/skops/blob/main/examples/plot_model_
+    card.py>`_ is an example of using skops to create a model card that can 
+    be used on the Hugging Face Hub.
+- Putting the Model Card on the Hub:
+    `Here <https://github.com/skops-dev/skops/blob/main/examples/plot_hf_hub.
+    py>`_ is an example of using skops to put a model card on the Hugging Face 
+    Hub.
+- Tabular Regression:
+    `Here <https://github.com/skops-dev/skops/blob/main/examples/plot_tabular
+    _classification.py>`_ is an example of using skops to serialize a tabular 
+    regression model and create a model card and a Hugging Face Hub repository.
+- Text Classification:
+    `Here <https://github.com/skops-dev/skops/blob/main/examples/plot_text_cl
```

Comment:
All the links here are to the .py files on GH. However, it is better to point them to the rendered documentation. E.g. for text classification, that would be: https://skops.readthedocs.io/en/stable/auto_examples/plot_text_classification.html